### ISSUE TITLE
PRODUCTION BUG: GRAPHQL Issue effecting BloodPac and BrainCommons

### ISSUE THREAD

**Issue by gneglur** [uc-cdis/peregrine#36]
1) fortuno [10:50 AM]
you have others in this channel too:
```{
   study(first:0, project_id:"bpa-USC_OPT1_T1", study_setup: "Genetic Analysis", with_path_to:{type: "read_group"}) {     
              submitter_id
  }
}```
(bloodpac)
```{
  demographic(with_path_to:{type:"case"}){
    submitter_id
  }
}```
(brain commons)

2) reubenonrye [10:48 AM]
@giangb @rudyard - some sample 'with_path_to' graphql from @fortuno: https://docs.google.com/document/d/1YD5MecoXnLLWH7Ii5KuX5ZRAx4xl7nd0iA2t2cuZ4O0/edit

**Comment by rudyardrichter** [comment]
Resolved by #38.

**Comment by rudyardrichter** [comment]
Time spent: ≈16 hours (including setting up namespace on dev cluster for testing).
